jest.config.js

```diff
@@ -5,9 +5,6 @@ module.exports = {
     setupFilesAfterEnv: ["./test-environment/jest.setup.js"],
     transformIgnorePatterns: ["node_modules/(?!(test-juggler)/)"],
     verbose: true,
-    "reporters": [
```

Comment:
PLease discard these white space changes, or add es-lint rule if they are needed

Reply:
Discarded and added `"no-trailing-spaces": 2` rule to eslint.